doc/radosgw/config-ref.rst

```diff
@@ -7,6 +7,11 @@ The following settings may added to the Ceph configuration file (i.e., usually
 settings may contain default values. If you do not specify each setting in the
 Ceph configuration file, the default value will be set automatically.
 
+Configuration variables set under the ``[client.radosgw.{instance-name}]``
+section will not apply to rgw or radosgw-admin commands without an instance-name
+specified in the command. Thus variables meant to be applied to all RGW
+instances or all radosgw-admin commands can be put into the ``[global]`` or the
```

Comment:
@alimaredia i think the knowledge of

> a setting in `[global]` section applies to all other sections

is orthogonal to what we are trying to convey here. if that is missing in the doc, we surely need to add it, but not to repeat it.